azure-pipelines.yml

```diff
@@ -18,124 +18,45 @@ resources:
-    displayName: Create config.h file
-  - powershell: |
-      echo "xerces3=1" | out-file -encoding ASCII $(ACE_ROOT)\bin\MakeProjectCreator\config\default.features
-      echo "ssl=1" | out-file -append -encoding ASCII $(ACE_ROOT)\bin\MakeProjectCreator\config\default.features
-      echo "versioned_namespace=1" | out-file -append -encoding ASCII $(ACE_ROOT)\bin\MakeProjectCreator\config\default.features
-    displayName: Create default.features file
-  - powershell: |
-      echo $(OptionalFeatures) | out-file -append -encoding ASCII $(ACE_ROOT)\bin\MakeProjectCreator\config\default.features
-    displayName: Add optional features ($(OptionalFeatures))
-    condition: and(succeeded(), ne(variables['OptionalFeatures'], ''))
-  - powershell: git clone -q --depth 1 git://github.com/DOCGroup/MPC.git $(MPC_ROOT)
-    displayName: git clone MPC
-  - powershell: perl $(ACE_ROOT)/bin/mwc.pl -type vs2019 $(TAO_ROOT)/TAO_ACE.mwc -workers 4
-    displayName: Run script mwc.pl on $(TAO_ROOT)/TAO_ACE.mwc
-  - powershell: perl $(ACE_ROOT)/bin/mwc.pl -type vs2019 $(ACE_ROOT)/tests/tests.mwc -workers 4
-    displayName: Run script mwc.pl on $(ACE_ROOT)/tests/tests.mwc
-  - task: VSBuild@1
-    displayName: Build solution TAO/TAO_ACE.sln
-    inputs:
-      solution: TAO/TAO_ACE.sln
-      platform: $(BuildPlatform)
-      configuration: $(BuildConfiguration)
-      maximumCpuCount: true
-  - task: VSBuild@1
-    displayName: Build solution ACE/tests/tests.sln
-    inputs:
-      solution: ACE/tests/tests.sln
-      platform: $(BuildPlatform)
-      configuration: $(BuildConfiguration)
-      maximumCpuCount: true
-
-- job: VisualStudio2017
-  timeoutInMinutes: 120
-  pool:
-    vmImage: vs2017-win2016
-  strategy:
-    matrix:
-      WChar:
-        BuildPlatform: x64
-        BuildConfiguration: Debug
-        vcpkgarch: x64-windows
-        vcpkglibdir: debug\lib
-        vcpkgpackages: 'openssl xerces-c[xmlch_wchar]'
-        OptionalFeatures: uses_wchar=1
-      Debug64:
-        BuildPlatform: x64
-        BuildConfiguration: Debug
-        vcpkgarch: x64-windows
-        vcpkglibdir: debug\lib
-        vcpkgpackages: openssl xerces-c
-      Release64:
-        BuildPlatform: x64
-        BuildConfiguration: Release
-        vcpkgarch: x64-windows
-        vcpkglibdir: lib
-        vcpkgpackages: openssl xerces-c
-  variables:
-    VCPKG_ROOT: $(Build.SourcesDirectory)\vcpkg
-    XERCESC_INCDIR: $(VCPKG_ROOT)\installed\$(vcpkgarch)\include
-    XERCESC_LIBDIR: $(VCPKG_ROOT)\installed\$(vcpkgarch)\$(vcpkglibdir)
-    SSL_INCDIR: $(VCPKG_ROOT)\installed\$(vcpkgarch)\include
-    SSL_LIBDIR: $(VCPKG_ROOT)\installed\$(vcpkgarch)\$(vcpkglibdir)
-  steps:
+      $Env:chocolateyUseWindowsCompression = $false
+      iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1')); choco feature disable --name showDownloadProgress
+      refreshenv
+      choco install -y 7zip
```

Comment:
7zip is default installed as far as I know, see https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/win/Vs2019-Server2019-Readme.md

Reply:
Oh, I didn't know that. This is my first time on this platform... :-\